app/screens/post_options/bindings/bindings.tsx

```diff
@@ -34,13 +35,31 @@ type Props = {
     };
 }
 
+const fetchBindings = (userId: string, channelId: string, teamId: string, setState: React.Dispatch<React.SetStateAction<AppBinding[] | null>>) => {
+    Client4.getAppsBindings(userId, channelId, teamId).then(
+        (allBindings) => {
+            const headerBindings = allBindings.filter((b) => b.location === AppBindingLocations.POST_MENU_ITEM);
+            const postMenuBindings = headerBindings.reduce((accum: AppBinding[], current: AppBinding) => accum.concat(current.bindings || []), []);
+            setState(postMenuBindings);
+        },
+        () => {/* Do nothing */},
+    ).catch(() => {/* Do nothing */});
```

Comment:
What error handling can we perform here if any?

Reply:
If there is any error happening here, the best we can do is log it. Anything else may be too disruptive for the user. Because of that, I would just leave it like that. If we weren't able to get the bindings, consider there are no bindings.